src/containers/checkout/delivery/delivery-type/mail-services/nova-poshta/nova-poshta-bottom.js

```diff
@@ -23,22 +28,26 @@ const NovaPoshtaBottom = ({ city }) => {
 
   useEffect(() => {
     city && dispatch(getNovaPoshtaWarehouse(city));
-    setInputValue('');
-  }, [dispatch, city]);
+    setDepartmentValue('');
+  }, [dispatch, city, setDepartmentValue]);
```

Comment:
Why you have a setter (`setDepartmentValue`) in dependencies, is there any sense to have it?